### PR TITLE
Frontend libs 9.5.0

### PR DESCRIPTION
**Pull request by goranalkovic-infinum**
More details in #803

### CODE REVIEW

**Review by iobrado — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,12 +4,26 @@ All notable changes to this project will be documented in this file.
...
 - `LinkInput` should now pop the suggestion panel only when focused into the input field
 - Fixed typo in `MultiSelect` story
+- Tweaked Gutenberg override classes for `NumberControl` so they don't override FE libs `NumberPicker`
+- Utility classes now use logical properties for paddings, margins, borders, widths, and heights
+- `IconToggle` is now `Toggle`, but the previous name will also work (for now)
+- Refreshed 3 UI icons
+- Updated dependencies
+- `ColorPicker` now has a slot for additional controls
 
 ## [9.4.2] - 2024-02-21
```
Comment (iruzevic) [inline CHANGELOG.md:29]: you are missing the change on the bottom of the page for links

File: package.json [hunk 2]
```diff
@@ -51,10 +51,10 @@
 		"@infinumjs/eslint-config-react-js": "^3.5.0",
```
Comment (iruzevic) [inline package.json:51]: you need to change the version in package.json for this release

**Review by iruzevic — CHANGES_REQUESTED** [review]

**Review by kancijan — APPROVED** [review]

**Review by iruzevic — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (goranalkovic-infinum) [reply CHANGELOG.md:29]: It's there, GH didn't detect it for some reason 🤷

File: package.json (hunk 2 above)
Comment (goranalkovic-infinum) [reply package.json:51]: It's there, GH didn't detect it for some reason 🤷

**Pull request merged**